app/assets/stylesheets/show.scss

```diff
@@ -20,6 +20,9 @@
     &.object-type-item {
       background-color: $illuminating-light;
     }
+    &.object-type-virtual-object {
+      background-color: $bay-light;
```

Comment:
Color chosen because it's in the brand guide and it's visually distinct from the other object type colors. Defer to @andrewjbtw and @astridu as to whether that's the best color to use here.

Reply:
I was not expecting to go all the way to a new page type with a pill. I thought we were just making a new facet.

Reply:
This commit adds both the facet and the show page pill. It doesn't need to, though, if we're not ready or if this isn't the design we want.

Reply:
I'm not opposed to going down this path, I just haven't thought through the implications. It does mean we would be able to take out the "download" links in the Contents section, which don't work for virtual objects since their content is metadata that links to content.